lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,6 +2,11 @@ module ActiveModel
   class Serializer
     module Adapter
       class Attributes < Base
+        def initialize(serializer, options = {})
+          super
+          @include_tree = IncludeTree.from_include_args(options[:include] || '*')
```

Comment:
maybe something like this?

```ruby
def self.included_associations(options)
  IncludeTree.from_include_args(options[:include] || '*')
end
```

and then `@include_tree = included_associations(options)` ?

idk